.gitignore

```diff
@@ -3,6 +3,7 @@ dist
 example*
 .gz
 coverage
+test/typings/**/*.js
```

Comment:
I'm ignoring the generated files from the TypeScript tests. There is an option in TypeScript to no emit any files (the `noEmit` option). However, I find it helpful sometimes to peek at the generated JavaScript to understand what it compiles to, which is particularly useful for JSX.